src/evaluate.py

```diff
@@ -128,7 +128,18 @@ def write_preds(tasks: Iterable[tasks_module.Task], all_preds, pred_dir, split_n
         preds_df = all_preds[task.name]
         # Tasks that use _write_glue_preds:
         glue_style_tasks = (tasks_module.ALL_NLI_PROBING_TASKS
-                            + tasks_module.ALL_GLUE_TASKS + ['wmt'])
+                            + tasks_module.ALL_GLUE_TASKS + ['wmt'] 
+                            + [ 'cola_npi_sup',
```

Comment:
sorry to be annoying, but could you define a custom list of tasks in `src/tasks/__init__.py` rather than listing them all here? Hopefully that should save you some work in other places if you need to reference this list again.

Reply:
I set a `ALL_COLA_NPI_TASKS` in `src/tasks/__init__.py` before, which went with automated task registration lines in `src/tasks/tasks.py`. But since now the latter has been removed, I fall back to hand-coding here in `src/tasks/__init__.py` too to keep consistency